.travis.yml

```diff
@@ -15,10 +15,9 @@ install:
   - unzip z3*.zip
   # NuGet is a little flakey in legacy TravisCI, use travis_retry command to retry the command if it fails
   - travis_retry nuget restore ${TRAVIS_SOLUTION}
+  - travis_retry nuget install -Version 3.10.0 NUnit.Console -OutputDirectory Source/packages/
```

Comment:
Why is it that this should be installed manually, rather than as a dependency that is restored by the previous line?

Reply:
This only concerns the Mono build and was already like this before. I just replaced the obsolete NUnit.Runners with NUnit.Console.

I guess the reason is that the previous line only restores the NUnit package needed to compile the unit test projects. This line installs the command line tool that runs the tests.

Reply:
Oh I see, and it’s right there in Travis’s own instructions too: https://docs.travis-ci.com/user/languages/csharp/#nunit

Just documenting for future reference then :-)